ocpp-v1_6/src/main/java/eu/chargetime/ocpp/JSONServer.java

```diff
@@ -30,16 +30,26 @@ of this software and associated documentation files (the "Software"), to deal
 import eu.chargetime.ocpp.model.Confirmation;
 import eu.chargetime.ocpp.model.Request;
 import eu.chargetime.ocpp.wss.WssFactoryBuilder;
+import org.java_websocket.drafts.Draft;
+import org.java_websocket.drafts.Draft_6455;
+import org.java_websocket.protocols.Protocol;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
-import javax.net.ssl.SSLContext;
+import java.util.Collections;
 import java.util.UUID;
 import java.util.concurrent.CompletionStage;
 
 public class JSONServer implements IServerAPI {
 
-    private WebSocketListener listener;
-    private Server server;
-    private FeatureRepository featureRepository;
+    private static final Logger logger = LoggerFactory.getLogger(JSONServer.class);
+
+    public final Draft draftOcppOnly =
+            new Draft_6455(Collections.emptyList(),
```

Comment:
Should be initiated in composite root (read the constructor).

Reply:
Initialization moved to constructor.